style.css

```diff
@@ -2,7 +2,7 @@
 @import 'map.css';
 @import 'galery.css';
 @import 'preloader.css';
-
+@import 'modal.css';
 @media (min-width: 1151px) {
```

Comment:
as i mentioned earlier - style.css serves us only for importing other styles - all the media queries should be in the same files they're unresponsive versions are